src/lsp/common.ts

```diff
@@ -4,20 +4,20 @@ import { LanguageServer } from "../types";
 import { SwiftMesonLspLanguageClient } from "./swift-mesonlsp";
 import { Uri } from "vscode";
 
+export function serverToClass(server: LanguageServer): any {
```

Comment:
Does it complain if you make the return value `typeof LanguageServerClient`?

Reply:
Yeah I feel like I tried to fix this before we merged the PR, and I couldn't figure it out then either.